### PR TITLE
BMD-2754 - Remove rds from module

### PR DESCRIPTION
**Pull request by bmwosz**
Change for new module path in git. RDS module change was not done yet in github.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -42,18 +26,6 @@ output "secret_key" {
   value = join("", data.aws_ssm_parameter.secret_key.*.value)
 }
 
-output "ecr_urls" {
```
Comment (Maciej-Gierwatowski) [inline outputs.tf:45]: Why removed this one?

File: main.tf [hunk 2]
```diff
@@ -175,14 +155,13 @@ resource "aws_autoscaling_group" "ecs_ec2_autoscalling_group" {
 }
 
 module "ecr" {
-  source = "git::https://github.com/BerlingskeMedia/bm.terraform-module.ecr"
```
Comment (Maciej-Gierwatowski) [inline main.tf:178]: How about refer to tag

File: outputs.tf (hunk 1 above)
Comment (bmwosz) [reply outputs.tf:45]: Good catch, my mistake

File: main.tf (hunk 2 above)
Comment (bmwosz) [reply main.tf:178]: I have added first tag to ecr module and updated this file also.

**Pull request merged**